package.json

```diff
@@ -12,19 +12,19 @@
     "plugin"
   ],
   "engines": {
-    "node": ">=4.0.0"
+    "node": ">=8.x.x"
   },
   "dependencies": {
     "boom": "5.x.x",
```

Comment:
Please bump all dependencies to latest, this doesn't make sense to stick to pre-v17 ones.

Reply:
@Marsup Deps updated